### PR TITLE
feat(color-select-panel): add color-update event

### PR DESCRIPTION
**Pull request by GaoNeng-wWw**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

1. add color-update event
2. fix color shake bug.

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/renderless/src/color-picker/utils/color.ts [hunk 1]
```diff
@@ -61,14 +62,17 @@ export default class Color {
     this.s = s
     this.v = v
     this.a = a
+    this.preH = h;
   }
-
   set({ h, s, v, a }: { h?: number; s?: number; v?: number; a?: number }) {
-    this.h = h ?? this.h
+    this.h = h ?? this.h;
```
Comment (zzcr) [inline packages/renderless/src/color-picker/utils/color.ts]: 这里是不是多了个分号？

File: packages/renderless/src/color-picker/vue.ts [hunk 2]
```diff
@@ -17,12 +17,11 @@ export const api = [
...
 export const renderless = (props, context, { emit }) => {
   const { modelValue, visible, predefine, size, history } = context.toRefs(props)
-  const hex = context.ref(modelValue.value ?? 'transparent')
-  const res = context.ref(modelValue.value ?? 'transparent')
-  const prev = context.ref(modelValue.value ?? 'transparent')
-  const triggerBg = context.ref(modelValue.value ?? 'transparent')
+  const pre = context.ref(modelValue.value ?? 'transparent');
+  const triggerBg = context.ref(pre.value ?? 'transparent');
+  const tmpColor = new Color(triggerBg.value ?? 'transparent');
+  const hex = context.ref(modelValue.value ?? 'transparent');
```
Comment (zzcr) [inline packages/renderless/src/color-picker/vue.ts]: 这里需要和整体代码风格保持一致，末尾不要加分号

Comment (GaoNeng-wWw) [reply packages/renderless/src/color-picker/vue.ts]: 已修改

File: packages/renderless/src/color-picker/utils/color.ts (hunk 1 above)
Comment (GaoNeng-wWw) [reply packages/renderless/src/color-picker/utils/color.ts]: 已修改

**Review by zzcr — APPROVED** [review]

**Pull request merged**